src/Compilers/CSharp/Portable/Symbols/NamespaceOrTypeSymbol.cs

```diff
@@ -253,9 +265,10 @@ public virtual ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name, int a
 
             NamedTypeSymbol? namedType = null;
 
-            ImmutableArray<NamedTypeSymbol> namespaceOrTypeMembers;
+            Debug.Assert(scope is NamespaceSymbol or NamedTypeSymbol);
             bool isTopLevel = scope.IsNamespace;
 
+            ImmutableArray<NamedTypeSymbol> namespaceOrTypeMembers;
```

Comment:
>ImmutableArray<NamedTypeSymbol> namespaceOrTypeMembers; [](http://example.com/codeflow?start=12&length=55)

Consider placing this declaration where it used to be and avoiding an unnecessary diff.

Reply:
done.